.github/CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ In order to contribute, you'll need to:
 1. Fork the repository.
 
 2. Create a branch, push your changes there. Don't forget to
-   {ref}`include news files for the changelog <_ansible_language_server_adding_changelog_fragments>`.
```

Comment:
This doesn't seem necessary

Reply:
It is. Prettier is escaping it on the other file if present and breaking sphinx building. Removing the underline from both makes them happy.

Reply:
So what you are saying is that prettier is not well-compatible with MyST.

FWIW this could be submitted separately. And explained.

Reply:
I think I just explained it and I do not think that splitting it even further is really needed.